grafeas-charts/Dockerfile

```diff
@@ -22,9 +22,9 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}
     tar -zxvf helm-${HELM_VERSION}-linux-amd64.tar.gz && \
     mv linux-amd64/helm /usr/local/bin/helm
 
-WORKDIR /go/src/github.com/grafeas/grafeas/grafeas-charts
-
 COPY . .
 
-RUN chmod +x helm_release.sh
+RUN echo $PWD
+RUN ls $PWD
+RUN chmod +x ./helm_release.sh
```

Comment:
This should be persisting from the outside repo (and git should store execute bits correctly). Is it not working?

Reply:
It didn't seem to be working, but at this point I'm unsure if it's because it couldn't find the script itself or it couldn't actually execute it (the error was ambiguous to me, and I didn't store the exact wording). I'll remove whatever is unneeded from the image once I get CloudBuild to build the image correctly.